.ci/scripts/7.0-upgrade.sh

```diff
@@ -5,16 +5,19 @@ test -z "$srcdir" && srcdir=.
 # shellcheck source=/dev/null
 . "${srcdir}/common.sh"
 
-export COMPOSE_ARGS="6.6 --release --force-build --build-parallel \
+docker build --build-arg GO_AGENT_BRANCH=v1.5.0 -t localtesting_6.6.2_opbeans-go docker/opbeans/go
```

Comment:
`--opbeans-go-agent-branch=1.x\` was replaced so, if this required too?

Reply:
really we want to use the current version, we tried with the branch but the branch would change in a few for sure so we've moved a fixed version that works, the 1.5.0 works.,